### ISSUE TITLE
Hyperparameter selection

### ISSUE THREAD

**Issue by sterling239** [NVIDIA/waveglow#128]
Hi! Has anyone experimented with hyperparameters, such as n_flows and n_mel_channels, in addition to n_channels in WN? As I understood correctly, the number of flow blocks increase a training time linearly, and n_mels is just conditioning detalization. So, what should I expect if I decrease the number of flows and increase n_mels?

**Comment by babua** [comment]
I no longer have access to the samples so I can't share, but from what I've tried, decreasing n_flows quickly degrades audio quality. I was trying to reduce model size and I couldn't find any meaningful tradeoff where reduction would be worth the loss in quality, even for small values of n_flow decrease.

If you increase n_mels I think that's going to make it even harder for the model to learn, because you're increasing dimensionality, and now you'd probably need more flows, not less.

**Comment by sterling239** [comment]
So, I've experimented with n_mels, and I conclude that n_mels slightly increase training speed (i've got good results at 350k steps with 12 batch size). Next i'll try to dance with WN parameters. Here I found, that receptive field of 3^8 points is much more than one group after "squeeze" operation in waveglow. That seems to be a bug in its architecture. So, I continue experimenting

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
